Source/Windows/ARM64EC/Module.cpp

```diff
@@ -567,10 +571,17 @@ NTSTATUS ProcessInit() {
   const uintptr_t KiUserExceptionDispatcherFFS = reinterpret_cast<uintptr_t>(GetProcAddress(NtDll, "KiUserExceptionDispatcher"));
   Exception::KiUserExceptionDispatcher = NtDllRedirectionLUT[KiUserExceptionDispatcherFFS - NtDllBase] + NtDllBase;
 
+  FEX_CONFIG_OPT(ProfileStats, PROFILESTATS);
+
+  if (IsWine && ProfileStats()) {
+    StatAllocHandler = fextl::make_unique<FEX::Windows::StatAlloc>(FEXCore::Profiler::AppType::WIN_ARM64EC);
+  }
   return STATUS_SUCCESS;
 }
 
-void ProcessTerm(HANDLE Handle, BOOL After, NTSTATUS Status) {}
+void ProcessTerm(HANDLE Handle, BOOL After, NTSTATUS Status) {
+  StatAllocHandler.reset();
```

Comment:
Can be dropped

Reply:
I think it can be done on the wine side on exit with my patch - but there's a lot of crashing cases where a process can exit without any exit handlers being called in general so it's probably best if mangohud can clean these up or something? It would be very nice to avoid having sigkill leak memory

Reply:
I'm more just concerned that this could be enabled by people that don't have something in place to cleanup afterwards, nothing in the option description suggests it's really quite likely to leak resources (a lot of things will launch subprocesses that end up crashing but work regardless) seems like something fexserver could do actually?

Reply:
Potentially FEXServer could do it on the Linux side, but since arm64ec/wow64 fex doesn't use FEXServer, it wouldn't work there.

Reply:
I think in the case you already have a Linux fex installation it might be fine? Though yeah it would have to poll shm files